dvc/tree/gdrive.py

```diff
@@ -345,29 +345,12 @@ def _gdrive_shared_drive_id(self, item_id):
         return item.get("driveId", None)
 
     @_gdrive_retry
-    def _gdrive_upload_file(
-        self,
-        parent_id,
-        title,
-        no_progress_bar=False,
-        from_file="",
-        progress_name="",
-    ):
+    def _gdrive_upload_fobj(self, fobj, parent_id, title):
```

Comment:
This is a bit unclear, internally pydrive2 uses `MediaIoBaseUpload`
https://github.com/iterative/PyDrive2/blob/ee368315132bd31bb643bf3d8af088cc512e39f3/pydrive2/files.py#L776-L778

the initializer of this wrapper says chunk size should be lower than 5MB
https://googleapis.github.io/google-api-python-client/docs/epy/googleapiclient.http.MediaIoBaseUpload-class.html#__init__ 

though the default chunk size is 100 MB (if I am not missing anything) (line #4721 https://googleapis.github.io/google-api-python-client/docs/epy/googleapiclient.http-pysrc.html#MediaIoBaseUpload.__init__

cc: @shcheklein

Reply:
Yes, information is confusing, probably they forgot to update the comment, or it should be interpreted in some other way. I see for example, that `rclone` has 8MB - https://forum.rclone.org/t/google-drive-and-optimal-drive-chunk-size/1186/11 ... and people mention 256M when they deal with very large files. I think it would be a good option to have, and we might indeed consider making it smaller by default to reduce memory pressure.

Reply:
we use `64 MiB` on all other providers, though I think there is no way (as of now) to pass the chunk_size into `MediaIoBaseUpload`. Maybe we should consider adding it as a feature to `pydrive2`?